data/watch.yml

```diff
@@ -1,151 +1,438 @@
 watch:
-  - name: State of the OCaml Platform 2020
```

Comment:
Can we sort the data to improve the diff and make sure we aren't tempering with already scraped data?